niceman/formats/tests/files/nicemanspec1_v0.0.1.yml

```diff
@@ -30,6 +30,7 @@ distributions:
        sha256: 0619d49952775fd1d9c1453aa2a065b876ec016e1dbcead09a90e44d1d82c561
        sha1: 470747b71b367b1bfa472ad294a51f44cfba874b
        md5: 17ba9548d5f3937431dab210b1a97aff
+       install_date: '2017-11-29 15:01:38+00:00'
```

Comment:
per my comment above, `install_date` isn't to be used here -- the `date` from apt source should be.